File: app/src/main/java/me/echeung/moemoekyun/viewmodel/AuthViewModel.java
package me.echeung.moemoekyun.viewmodel;

import androidx.databinding.Bindable;

import androidx.databinding.library.baseAdapters.BR;

public class AuthViewModel extends BaseViewModel {

    private boolean showRegister = false;

    @Bindable
    public boolean getShowRegister() {
        return showRegister;
    }

    public void setShowRegister(boolean showRegister) {
        this.showRegister = showRegister;
        notifyPropertyChanged(BR.showRegister);
    }

}

File: app/src/main/java/me/echeung/moemoekyun/viewmodel/AuthViewModel.kt
package me.echeung.moemoekyun.viewmodel

import androidx.databinding.Bindable
import androidx.databinding.library.baseAdapters.BR

class AuthViewModel : BaseViewModel() {

    @get:Bindable
    var showRegister = false
        set(showRegister) {
            field = showRegister
            notifyPropertyChanged(BR.showRegister)
        }

}
